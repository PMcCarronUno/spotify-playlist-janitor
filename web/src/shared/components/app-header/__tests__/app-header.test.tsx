import { render } from "@testing-library/react";

import { AppHeader } from "..";

describe("<AppHeader />", () => {
  let container: HTMLElement;

  beforeEach(() => {
    ({ container } = render(<AppHeader />));
  });

  it("should render AppHeader component", () => {
    expect(container.firstChild).toMatchSnapshot();
  });
});
