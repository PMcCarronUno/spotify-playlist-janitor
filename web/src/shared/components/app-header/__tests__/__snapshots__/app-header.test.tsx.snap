// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppHeader /> should render AppHeader component 1`] = `
<div
  class="sc-jTQDJr buqzbX"
>
  <div
    class="sc-guDMob jLqGJD"
  >
    Spotify Playlist Janitor
  </div>
</div>
`;
